Tests/MapboxMapsTests/Foundation/MapViewTests.swift

```diff
@@ -252,23 +252,60 @@ final class MapViewTests: XCTestCase {
     }
 
     func testDisplayLinkPausedWhenAppWillResignActive() {
+        displayLink.$isPaused.setStub.reset()
+
         notificationCenter.post(name: UIApplication.willResignActiveNotification, object: nil)
 
         XCTAssertEqual(displayLink.$isPaused.setStub.invocations.map(\.parameters), [true])
     }
 
+    func testDisplayLinkPausedWhenAppDidEnterBackground() {
+        displayLink.$isPaused.setStub.reset()
+
+        notificationCenter.post(name: UIApplication.didEnterBackgroundNotification, object: nil)
+
+        XCTAssertEqual(displayLink.$isPaused.setStub.invocations.map(\.parameters), [true])
+    }
+
     func testReleaseDrawablesInvokedWhenAppDidBecomeActive() {
         notificationCenter.post(name: UIApplication.didEnterBackgroundNotification, object: nil)
 
         XCTAssertEqual(metalView.releaseDrawablesStub.invocations.count, 1)
     }
 
-    func testDisplayLinkResumedWhenAppDidBecomeActive() {
+    func testDisplayLinkResumedWhenAppDidBecomeActiveOnIOS12() throws {
```

Comment:
[Discussion] Should we extract all iOS12 test functions to the separate extension or even a separate file (still an extension)?

Reply:
I don't think we have so much of them to warrant that